politeiawww/user.go

```diff
@@ -115,3 +117,34 @@ func (b *backend) ProcessEditUser(eu *v1.EditUser, user *database.User) (*v1.Edi
 
 	return &v1.EditUserReply{}, nil
 }
+
+// ProcessProposalAccessTime returns an array of the access times for
+// all proposals for an user
+func (b *backend) ProcessProposalAccessTime(email string, token string) (*www.ProposalAccessTimeReply, error) {
+	log.Tracef("ProcessProposalAccessTime")
+	user, err := b.db.UserGet(email)
+	if err != nil {
+		return nil, err
+	}
+	return &www.ProposalAccessTimeReply{
+		ProposalAccessTime: user.ProposalAccessTimes[token],
+	}, nil
+}
+
+// ProcessNewProposalAccessTime inserts new access for some proposal by a given user
+func (b *backend) ProcessNewProposalAccessTime(email string, token string) (*www.ProposalAccessTimeReply, error) {
+	log.Tracef("ProcessProposalAccessTime")
+	user, err := b.db.UserGet(email)
```

Comment:
You don't need to do this.  You already look up the user in `handleNewProposalAccessTime `.